internal/cli/atlas/deployments/diagnostics.go

```diff
@@ -105,11 +105,10 @@ func DiagnosticsBuilder() *cobra.Command {
 			}
 
 			w := cmd.OutOrStdout()
-			opts.podmanClient = podman.NewClient(log.IsDebugLevel(), w)
 
 			return opts.PreRunE(
 				opts.InitOutput(w, ""),
-				opts.InitStore(opts.podmanClient),
+				opts.InitStore(cmd.Context(), podman.NewClient(log.IsDebugLevel(), w)),
```

Comment:
`w` here is not right, has to be stderr, try `log.Writer()`

Reply:
alternatively you can remove these params and call `log.Debug...` inside `podman.Client`

Reply:
I think we won't even need to init podman anymore then